.github/workflows/ci.yml

```diff
@@ -157,7 +157,7 @@ jobs:
           path: nuget/bin-macos
 
       - name: Build NuGet package
-        run: nuget pack OpenTelemetry.AutoInstrumentation.nuspec -Properties NoWarn=NU5100,NU5123,NU5128
+        run: nuget pack OpenTelemetry.AutoInstrumentation.nuspec -Properties NoWarn=NU5100,NU5123,NU5128,NU1608
```

Comment:
Process and Runtime packages will be released today/tommorow. Alan is working on it.
I am blocking PR until it will be available,

Reply:
Upgraded the versions for `OpenTelemetry.Instrumentation.Runtime` and `OpenTelemetry.Instrumentation.Process`. 

Synced the rename and removal of DLLs originating from OTel SDK changes.